README.md

```diff
@@ -34,8 +36,8 @@ pip install urllib3
 - Open the 'script starter.bat' to launch the script.
 - Go to the folder with this program. Then press Shift + right click mouse button anywhere in the folder window and select the 'Open command window here' option from the context menu. Then enter 'python fimfiction_stories_downloader.py' and click Enter.
 - Open the 'command line' and go to the directory with the program files using the 'cd' command. Then enter 'python fimfiction_stories_downloader.py' to launch the script. You can also add the full path after 'python'.
-5. Answer a few simple questions to indicate what you want to download. The program uses the card view to count the number of stories and pages (except for search results, popular stories etc.) so you have to keep that in mind when you paste links into it.
-6. Stories will be saved to the folder 'downloaded stories' that will be automatically created in the same folder where you put the program.
+5. Answer a few simple questions to indicate what you want to download. The program uses the card view to count the number of stories and pages so you have to keep that in mind when you paste links into it.
+6. Stories will be saved to the folder 'downloaded_stories' that will be automatically created in the current working directory.
```

Comment:
I changed the phrasing to "current working directory" because technically the 'downloaded_stories' directory has always been created in the directory from which the user called the program.